### ISSUE TITLE
Is self hosted and ecrypted jitsi blockable by an ISP?

### ISSUE THREAD

**Issue by ChechyLevas** [jitsi/docker-jitsi-meet#186]
I am able to get jitsi working with devices on my LAN able to send/receive audio and video.
However, for a device trying to connect from the internet, it partially works. The user can join the room and participate in text chat, but can neither send nor receive audio/video.

I live in a country where traditional voip is blocked by the ISP. My hope was that jitsi would be immune to this. Is that hope unfounded? Does encrypted jitsi traffic look like any other voip traffic? Any ideas to get around this?

**Comment by saghul** [comment]
Media traffic uses RTP which I guess ISPs may lock just by looking at the fact that you’re sending tons of small UDP packets, even if they cannot decrypt them.

Maybe try using TURN with TCP but your expendiendo will be degraded because media over TCP is not great.

**Comment by ChechyLevas** [comment]
I think you are right about UDP being restricted. Based on your resposne I found [this jisti-videobridge document](https://github.com/jitsi/jitsi-videobridge/blob/master/doc/tcp.md) which says: 
_ICE/TCP is not the recommended way to deal with clients connecting from networks where UDP traffic is restricted. The recommended way is to use jitsi-videobridge in conjunction with a TURN server._

Is that applicable in this case? How do I set this up?
